packages/patternfly-4/react-core/src/components/Select/Select.test.tsx

```diff
@@ -22,10 +22,10 @@ class User implements SelectOptionObject {
 }
 
 const selectOptions = [
-  <SelectOption value="Mr" key="0" />,
-  <SelectOption value="Mrs" key="1" />,
-  <SelectOption value="Ms" key="2" />,
-  <SelectOption value="Other" key="3" />
+  <SelectOption value="Mr" key="00" />,
+  <SelectOption value="Mrs" key="01" />,
+  <SelectOption value="Ms" key="02" />,
+  <SelectOption value="Other" key="03" />
```

Comment:
Curious about the key values, is the change necessary?

Reply:
There was problem with tests complaining about same keys.

Reply:
ok, thanks for the explanation